images/linux/toolsets/toolset-2004.json

```diff
@@ -196,7 +196,8 @@
         ]
     },
     "brew": [
-        {"name": "zstd"}
+        {"name": "zstd"},
+        {"name": "kotlin"}
```

Comment:
It's not a good idea to install kotlin via brew as it depends on "openjdk" package, which will be added to the path and can break our java tools installation

Reply:
And moreover, brew installation consumes much more space on disk due to the openjdk and related packages installation